src/signals/my-incidents/components/IncidentsDetail/IncidentsDetail.tsx

```diff
@@ -0,0 +1,86 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+import { useMemo } from 'react'
+
+import { Paragraph } from '@amsterdam/asc-ui'
+
+import { StyledHeading } from '../../pages/styled'
+import type { MyIncident } from '../../types'
+import {
+  FormTitle,
+  StyledImage,
+  ImageWrapper,
+  StyledBacklink,
+  StyledLink,
+  Wrapper,
+} from './styled'
+
+type Props = {
+  token: string
+  setShowMap: (show: boolean) => void
+  incidentsDetail?: MyIncident
+}
+
+export const IncidentsDetail = ({
+  token,
+  setShowMap,
+  incidentsDetail,
+}: Props) => {
+  const answersGebeurtHetVaker = useMemo(() => {
+    return incidentsDetail?.extra_properties?.find(
+      (property) => property.id === 'extra_personen_overig_vaker_momenten'
+    )?.answer
+  }, [incidentsDetail?.extra_properties])
+
+  if (!incidentsDetail) {
+    return null
+  }
+
+  const attachments = incidentsDetail?._links?.['sia:attachments']
+
+  return (
+    <>
+      <StyledBacklink to={`/mijn-meldingen/${token}`}>
+        Mijn meldingen
+      </StyledBacklink>
+
+      <header>
+        <StyledHeading>{`Meldingsnummer: ${incidentsDetail._display}`}</StyledHeading>
+      </header>
+      <Wrapper>
+        <FormTitle>Omschrijving</FormTitle>
+        <Paragraph strong>{incidentsDetail?.text}</Paragraph>
+      </Wrapper>
+
+      <Wrapper>
+        {attachments && (
+          <FormTitle>Foto{attachments.length > 1 && "'s"}</FormTitle>
+        )}
+        {attachments?.map((attachment, index) => (
+          <ImageWrapper key={attachment.href + index}>
+            <StyledImage src={attachment.href} />
+          </ImageWrapper>
+        ))}
+      </Wrapper>
+
+      <Wrapper>
+        <FormTitle>Locatie</FormTitle>
+        <Paragraph strong style={{ marginBottom: 0 }}>
```

Comment:
I removed this margin only once. That's why it seemed more logically to me to remove it instead of making a styled component.